### PR TITLE
Add pagy for followers route

### LINKED ISSUE #8573: Add pagy for followers route
**Please describe the desired behavior.**
Pagy is a pagination gem used for its peak performance over other pagination tools. As mentioned #8426  we have been trying to migrate our codebase from will_paginate to pagy gem and would love your help with it! :tada:  

Here is how you may implement pagy for this: - 

- Select action `followers` under `users_controller`
- Since here pagination is already implemented using `will_paginate` as `.paginate(page: params[:page], per_page: 10)` you can remove this line so that the method now looks like 

    `@users = @user.followers`

- Now add the pagy method in this action as - 
`@pagy, @users = pagy(@user.followers, items: 10)`

Here 10 represents the items to be displayed on each page. This is the most basic way we can use pagy over any action: -
` @pagy, @records = pagy(Product.some_scope)`

- Now move to the html code which is generated for this controller `users/show_follow.html` 
- Search for the `will_paginate` line and replace it with - 

`<% if @pagy %>
        <%= raw pagy_bootstrap_nav @pagy %>
<% else %>
        <%= will_paginate @users, renderer: WillPaginate::ActionView::BootstrapLinkRenderer %>
 <% end %>`

- This will now check if `@pagy` variable is available or not and then if present will generate pagination using pagy.
- Awesome, create a pull request for the same and add screenshots if relevant! 

**Additional context (optional)**
In case of queries please checkout the following links  for help: - 
https://ddnexus.github.io/pagy/how-to
https://ddnexus.github.io/pagy/migration-guide
https://github.com/publiclab/plots2/pull/8428
https://github.com/publiclab/plots2/pull/8326

### PR DESCRIPTION
**Pull request by KarenEfereyan**
This PR fixes #8573 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x ] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by KarenEfereyan** [comment]
@Tlazypanda pls give me some more time. I have to install the dependenciess and run the tests. Thats why the Travis ci test is failing. I'll get to it as so soon as possible

**Review by Tlazypanda — APPROVED** [review]
Hey @KarenEfereyan approved good work 🎉 actually the test sometimes fail and we need to retrigger it no worries!

**Comment by KarenEfereyan** [comment]
> Thanks for opening this pull request! This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
> `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
> Also please refer [here](https://github.com/publiclab/plots2/issues/3840) for installation help 💿
> There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄
> It would be great if you can tell us your Twitter handle so we can thank you properly?

twitter.com/EfereyanK

**Comment by KarenEfereyan** [comment]
> Hey @KarenEfereyan approved good work 🎉 actually the test sometimes fail and we need to retrigger it no worries!

Makes me so happy. Thank you

File: app/views/users/show_follow.html.erb [hunk 1]
```diff
@@ -17,7 +17,9 @@
         </tr>
       <% end %>
     </table>
-    <%= will_paginate @users, renderer: WillPaginate::ActionView::BootstrapLinkRenderer %>
+   <% if @pagy %> <%= raw pagy_bootstrap_nav @pagy %>
```
Comment (cesswairimu) [inline app/views/users/show_follow.html.erb:20]: Hey @KarenEfereyan could you please separate this to two lines like so, 
```
<% if @pagy %>
  <%= raw pagy_bootstrap_nav @pagy %> 
```
This is mainly for readability purposes thanks

**Review by cesswairimu — CHANGES_REQUESTED** [review]

File: app/views/users/show_follow.html.erb (hunk 1 above)
Comment (KarenEfereyan) [reply app/views/users/show_follow.html.erb:20]: I will. Thank you

**Comment by KarenEfereyan** [comment]
@cesswairimu I have made the necessary changes, Kindly review

**Review by Tlazypanda — APPROVED** [review]
Hey @KarenEfereyan thanks for making this more readable now :tada: :clap:

**Comment by KarenEfereyan** [comment]
> Hey @KarenEfereyan thanks for making this more readable now 🎉 👏

You are welcome. Thank you for helping me make it better. I await merging

**Comment by KarenEfereyan** [comment]
> > Hey @KarenEfereyan thanks for making this more readable now 🎉 👏
> 
> You are welcome. Thank you for helping me make it better. I await merging

I just realized there's a merge conflict. I'll fix it asap

**Comment by KarenEfereyan** [comment]
@cesswairimu @Tlazypanda @jywarren i have made some final changes. My PR had conflicts earlier due to some updated i hadn't incorporated. But i've resolved the conflicts now. Can you please review?

**Review by cesswairimu — APPROVED** [review]

**Comment by cesswairimu** [comment]
Thanks for working on this @KarenEfereyan  :tada: and thanks @Tlazypanda  for the review :100:

**Comment by KarenEfereyan** [comment]
> Thanks for working on this @KarenEfereyan 🎉 and thanks @Tlazypanda for the review 💯

Thanks @cess and @Tlazypanda

**Pull request merged**